README.md

```diff
@@ -265,6 +265,154 @@ end)
 
 10. Install the dependencies below.
 
+11. (OPTIONAL) If you want to use [qb-doorlock](https://github.com/qbcore-framework/qb-doorlock) to lock doors inside your shell then add the following code into `qb-doorlock > client > main.lua > around line 812` under `RegisterKeyMapping('remotetriggerdoor', Lang:t("general.keymapping_remotetriggerdoor"), 'keyboard', 'H')`
+```lua
+function generateRandomString(length)
+    local characters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    local randomString = ""
+
+    for i = 1, length do
+        local randomIndex = math.random(1, #characters)
+        randomString = randomString .. string.sub(characters, randomIndex, randomIndex)
+    end
+
+    return randomString
+end
+
+RegisterNetEvent('qb-doorlock:client:addNewDoorPS', function()
+	canContinue = false
+	hideNUI()
+	if not Config.SaveDoorDialog then doorData = {} end
+	local dialog = exports['qb-input']:ShowInput({
+		header = "Add a new doorlock to your shell!",
+		submitText = Lang:t("general.submit_text"),
+		inputs = {
+			{
+				text = Lang:t("general.doortype_title"),
+				name = "doortype",
+				type = "select",
+				options = {
+					{ value = "door", text = Lang:t("general.doortype_door") },
+					{ value = "double", text = Lang:t("general.doortype_double") }
+				},
+				default = Config.SaveDoorDialog and doorData.doortype,
+			}
+		}
+	})
+	if not dialog or not next(dialog) then canContinue = true return end
+	doorData = dialog
+
+	local propertyId = exports['ps-housing']:GetPropertyId()
+	local randomchars = generateRandomString("4")
+	local uniqueid = "propertyid_" ..propertyId.. "_door_" ..randomchars
+	doorData.configfile = "PS ( " ..PlayerData.citizenid.. " )"
+	doorData.dooridentifier = uniqueid
+	doorData.doorlabel = uniqueid
+	doorData.cid = PlayerData.citizenid
```

Comment:
Can all of these doorData fields be camelCased or is there something else that requires them to be all lowercase?

Reply:
thats on qb-doorlock

Reply:
lame, but ok ;)